build/lib/FaceJoy.svg

```diff
@@ -0,0 +1 @@
+<svg aria-hidden="true" class="svg-icon iconFaceJoy" width="18" height="18" viewBox="0 0 18 18"><path d="M12.58 10.49a.86.86 0 0 0-.82-.36c-1.84.21-3.7.21-5.53 0a.87.87 0 0 0-.89 1.25 4 4 0 0 0 3.66 2 4 4 0 0 0 3.66-2 .87.87 0 0 0-.08-.89z"/><path d="M9 17A8 8 0 1 1 9 1a8 8 0 0 1 0 16zm0-2A6 6 0 1 0 9 3a6 6 0 0 0 0 12z"/><path d="M7.71 8.25a.56.56 0 0 0 .54-.59v-.72A1.37 1.37 0 0 0 7 5.5a1.34 1.34 0 0 0-1.25 1.44v.72a.56.56 0 0 0 .54.59h1.42zm4 0a.56.56 0 0 0 .54-.59v-.72A1.37 1.37 0 0 0 11 5.5a1.34 1.34 0 0 0-1.25 1.44v.72a.56.56 0 0 0 .54.59h1.42z" fill="#000"/></svg>
```

Comment:
This shouldn't have a fill of `#000` in it 🤔 . I think it's related to some strange grouping in the Figma doc.